storage.js

```diff
@@ -0,0 +1,31 @@
+const formStorage = document.querySelector('form');
+const nameStorage = document.querySelector('name');
+const emailStorage = document.querySelector('email');
+const messageStorage = document.querySelector('message');
```

Comment:
- [ ] Your items are not being stored properly. Please make sure you are getting the correct items from your form. It looks like you are not getting the correct name and message from your HTML. You can use `console.log` on those items to check that.